buildSrc/subprojects/plugins/plugins.gradle.kts

```diff
@@ -5,7 +5,7 @@ dependencies {
     implementation(project(":kotlinDsl"))
     implementation(project(":versioning"))
     implementation(project(":performance"))
-    implementation("org.pegdown:pegdown:1.6.0")
+    implementation("com.vladsch.flexmark:flexmark-all:0.34.48")
```

Comment:
I think we shouldn't have the version number in three different places. Shall we add the dependency to buildSrc?

Moreover, it seems like flexmark `0.34.56` is available. Any reason why we are using `.48`?

Reply:
Interestingly seems like `flexmark` dependency is only necessary in only one place. I removed two other declarations and everything seems to work.

I've updated to `0.34.56`. I used `.48` because that's in the official document and didn't search for latest.